defaults/main.yml

```diff
@@ -42,4 +42,6 @@ openio_redis_auth_pass: ""
 openio_redis_gridinit_file_prefix: ""
 openio_redis_gridinit_dir: "/etc/gridinit.d/{{ openio_redis_namespace }}"
 openio_redis_provision_only: false
+
+openio_redis_configuration_splitted: true
```

Comment:
this is an irregular verb, so just "s/splitted/split/g"

Reply:
done